build-farm/platform-specific-configurations/windows.sh

```diff
@@ -202,28 +202,15 @@ then
     # NASM required for OpenSSL support as per #604
     export PATH="/cygdrive/c/Program Files/LLVM/bin:/usr/bin:/cygdrive/c/openjdk/nasm-$OPENJ9_NASM_VERSION:$PATH"
   else
-    TOOLCHAIN_VERSION="2013"
+    TOOLCHAIN_VERSION="2017"
+    export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --disable-ccache"
     if [ "${JAVA_TO_BUILD}" == "${JDK8_VERSION}" ]
     then
-      export BUILD_ARGS="${BUILD_ARGS} --freetype-version 2.5.3"
+      export BUILD_ARGS="${BUILD_ARGS} --freetype-version 2.8.1"
```

Comment:
I've just reverted to let's see if it can build. The main thing to note is that I had to apply the `freetype.vcproj` patch as per https://github.com/adoptium/jdk8u/blob/master/.github/workflows/freetype.vcxproj. Supposedly there is a backport that will fix that so I'm not sure whether you've already implemented that in the OpenJ9 source code?

Reply:
Hmm no that didn't pass, I think that the .proj file is for a later version of Freetype only

Reply:
I'd also prefer to see a more modern version of freetype used if possible.  We should actually really be compiling and shipping with the latest possible.

Reply:
I don't think OpenJ9 uses `freetype.vcxproj`, but, yes, I agree it's wise to favour modern versions.